src/Nethermind/Nethermind.AccountAbstraction/Source/PaymasterThrottler.cs

```diff
@@ -49,19 +49,16 @@ public class PaymasterThrottler : IPaymasterThrottler
                 )
             );
 
-        public PaymasterThrottler()
-        {
-            _opsSeen = new Dictionary<Address, uint>();
-            _opsIncluded = new Dictionary<Address, uint>();
+        public PaymasterThrottler() : this(true) { }
 
-            SetupAndStartTimer();
```

Comment:
why did you remove this?

Reply:
Actually this constructor now calls the other constructor from the same class, and this step is done there

Reply:
I think this way is better, since now there is only the constructor logic is concentrated in only one place